src/OneExplorer/OneExplorer.ts

```diff
@@ -543,6 +543,7 @@ class OneccRunner extends EventEmitter {
             .catch(value => {
               vscode.window.showWarningMessage(
```

Comment:
(Q) Is it your intention to use warning message instead of error message?

Reply:
👍 I created #713.

Reply:
How about merging this PR apart from discussion? 😄

Reply:
This should be fixed to error. I will do that in different PR. :-)

Reply:
Merge please :) /cc @Samsung/one-vscode_committers